README.md

```diff
@@ -1,6 +1,9 @@
-  <p align="center">
-  <img src="src/assets/logo.svg" height="42">
-  <h2 align="center">Openverse Browser Extension</h2>
+<img src="https://github.com/WordPress/openverse/raw/main/brand/banner.svg" width="100%"/>
+
+<p align="center">
+  <a href="https://github.com/orgs/WordPress/projects/3">Project Board</a> | <a href="https://make.wordpress.org/openverse/">Community Site</a> | <a href="https://make.wordpress.org/chat/">#openverse @ Slack</a> | <a href="https://make.wordpress.org/openverse/handbook/openverse-handbook/">Handbook</a> | <a href="https://www.figma.com/file/w60dl1XPUvSaRncv1Utmnb/Openverse-Releases?node-id=0%3A1">Releases mockups</a> | <a href="https://www.figma.com/file/GIIQ4sDbaToCfFQyKMvzr8/Openverse-Design-Library?node-id=0%3A1">Design Library</a>
+
+  <h1 align="center">Openverse Browser Extension</h2>
```

Comment:
Just a little typo here. The title (the `<h1></h1>` element) of the page is the breadcrumb (WordPress / openverse-...) so I think this has to be an `<h2>` element, or choose whatever you consider is more appropriate.
```suggestion
  <h2 align="center">Openverse Browser Extension</h2>
```

Reply:
Interesting! Our other repos use an h1 `#` in markdown. I will look into which is best

Reply:
@krysal I didn't understand. Considering that it's the Markdown files topmost heading, shouldn't it be `h1`? The rest of the GitHub UI should not be a factor in this.

Reply:
The Markdown is translated to HTML so I was saying it because, theoretically, is better to have just one `h1` element by page but looks like even GitHub doesn't follow this practice. I inspected the entry page to the repository and it repeats several times, so is not a big deal here.